.github/actions/libcxx/run-buildbot/action.yaml

```diff
@@ -0,0 +1,44 @@
+name: 'Run Buildbot for libc++'
+description: 'Invoke the run-buildbot script with the specified parameters'
+
+inputs:
+  configuration:
+    description: 'the configuration to build'
+    required: true
+  cxx:
+    description: 'C++ compiler'
+    required: true
+  cc:
+    description: 'C compiler'
+    required: true
+  enable_clang_tidy:
+    description: 'enable clang-tidy'
+    required: false
+    default: 'OFF'
+  enable_std_modules:
+    description: 'enable std modules'
+    required: false
+    default: 'OFF'
+
+runs:
+  using: "composite"
```

Comment:
What does this do?

Reply:
Do you have a link to the documentation for this kind of action?  I have not seen this before.